### ISSUE TITLE
Export method not working on Page object ([__NSSingleObjectArrayI name]: unrecognized selector)

### ISSUE THREAD

**Issue by legranddamien** [BohemianCoding/SketchAPI#101]
## Summary

I'm trying to export a Page with the new API (Sketch 49). It works well when giving an artboard to the export method. But when giving a Page object it crash (the script, not the app). 

http://developer.sketchapp.com/reference/api/#export

## Steps to Reproduce:

Create a plugin to test with this script :

```js
var sketch = require('sketch/dom');

export default function(context) {
    
    var document = sketch.Document.getSelectedDocument();

    //I know I have 2 Pages on my test document in Sketch
    log(document.pages);

    for (var i = 0; i < document.pages.length; i++) {
        var page = document.pages[i];
        
        log(page);
        log(page.name);

        sketch.export(page, {
            overwriting: true,
            output: "~/Desktop/test_output"
        });
    }
}

// Logs
// 
// // Pages
// (
//     "<null>",
//     "<null>"
// )

// // Page [0]
// (null) // page
// screen_1 // page.name

// // sketch.export error
// -[__NSSingleObjectArrayI name]: unrecognized selector sent to instance 0x60800001a150

```

## Expected Results
An image should be created representing the full page

## Actual Results:

An error : 

`-[__NSSingleObjectArrayI name]: unrecognized selector sent to instance 0x60800001a150`

## Notes 
It's weird, because when logging a page, `null` is printed, but `page.name`, print the name.

**Comment by mathieudutour** [comment]
Good catch! It will be fixed in 49.1.

> It's weird, because when logging a page, null is printed, but page.name, print the name.

Yeah I'm not sure why. I'm working on improving the logs in general, it will be a lot better really soon! In the meantime, you can use `page.toJSON()` and log this

**Comment by mathieudutour** [comment]
(also, thanks for the nice bug report 💯 )
